tests/driver/driverlib/lib.rs

```diff
@@ -28,6 +29,8 @@ impl TestCase {
 pub fn collect_test_cases(sub_folders: &str) -> std::io::Result<Vec<TestCase>> {
     let mut results = vec![];
 
+    let all_styles = ["fluent", "material", "cupertino", "cosmic"];
```

Comment:
I realize I don't know how to test if qt is available, here in the driver / build.rs. It seems like the driver lib would have to depend on I-slint-backend-qt and then we look for `DEP_I_SLINT_BACKEND_QT_SUPPORTS_NATIVE_STYLE`?

Reply:
> It seems like the driver lib would have to depend on i-slint-backend-qt and then we look for DEP_I_SLINT_BACKEND_QT_SUPPORTS_NATIVE_STYLE

Yes.  Which is kind of fine because without the default feature (the "enable" one) the backend is pretty much empty.

You can try to read `SLINT_DEFAULT_STYLE.txt` in the right target directory.
see get_native_style in style_loader.

It's basically $OUT_DIR/../../SLINT_DEFAULT_STYLE.txt  and if the content is "qt" one should use that. But that means Qt will only be tested on linux  but that's fine.

But both solution will give the wrong answer when compiling the tests with `--target=`, because the driverlib is always a build dependency.

Reply:
I've added reading `SLINT_DEFAULT_STYLE.txt`. Oddly, it doesn't work on my Linux. After installing Qt, for example `cargo run -p gallery` comes up with the Qt style, but that text file has "fluent" in it.

Reply:
Ah, after the gallery builds, it contains qt. But when the rust driver is built, the contents are overwritten and it's back to fluent.

Reply:
Ahh, that would explain it. What shall we do? Take this in as-is? Add the backend-qt feature dependency? (asking for my laptop that'll be offline soon ;)

Reply:
I think we can take it as it.

We could add an explicit dependency to the Qt backend from the rust driver as well